README.md

```diff
@@ -122,6 +122,12 @@ B attributes will be merged into A.
 
 B is also destroyed after the merge has been successful.
 
+## Testing
+
+In the terminal, run `sudo mongod` to start the mongoDB connection
```

Comment:
It's a simpler way of starting mongoDB if you don't test with mongo often. If you want to run without `sudo` you can run `sudo chmod -R 777 /data/db` first, then from then on, you'll only need to run `mongod`.